OTRRouting/OTRRoutingController.m

```diff
@@ -22,6 +23,16 @@ - (instancetype _Nonnull)init {
   self.baseUrl = @"https://valhalla.mapzen.com/route?";
   self.urlSessionManager = [NSURLSession sessionWithConfiguration:[NSURLSessionConfiguration defaultSessionConfiguration]];
   self.urlQueryComponents = [[NSMutableArray alloc] init];
+  self.locale = [NSLocale currentLocale];
+  return self;
+}
+
+- (instancetype _Nonnull)initWithSessionManager:(NSURLSession*  _Nonnull)session {
```

Comment:
@sarahlensing I'm not sure if this is a common pattern in obj-C but can `init` just call `initWithSessionManager` and pass the default session as a param?